lib/rules/space-before-function-parentheses.js

```diff
@@ -115,3 +115,25 @@ module.exports = function(context) {
         "FunctionExpression": validateSpacingBeforeParentheses
     };
 };
+
+module.exports.schema = [
```

Comment:
Yeah, I added schemas to the deprecated rules too because it was easy enough once I was in the flow, and, like you said, it doesn't hurt. It also made it easier to know I didn't miss any rules.